recipe/meta.yaml

```diff
@@ -76,6 +76,7 @@ requirements:
   run_constrained:
     - khronos-opencl-icd-loader  ==9999999999  # [osx and opencl_impl == 'apple']
     - ocl_icd_wrapper_apple      ==9999999999  # [osx and opencl_impl == 'apple']
+    - python <3.9.9  # [py == 39 and win]
```

Comment:
Can you move this to `run`?

Reply:
Is that in addition to listing just `python` under `run`, or instead of it?

Reply:
In addition

Reply:
That didn't work.  It still installed 3.9.9.